frontend/src/components/common/RichTextField.tsx

```diff
@@ -0,0 +1,392 @@
+import {
+  convertFromRaw,
+  convertToRaw,
+  Editor,
+  EditorState,
+  getDefaultKeyBinding,
+  KeyBindingUtil,
+  Modifier,
+  RichUtils,
+  SelectionState,
+} from "draft-js";
+import React, { FunctionComponent, useState } from "react";
+import {
+  VStack,
+  HStack,
+  ButtonGroup,
+  Select,
+  Image,
+  Divider,
+  Box,
+} from "@chakra-ui/react";
+import "draft-js/dist/Draft.css";
+import Bold from "../../assets/RichTextField/B.svg";
+import Italic from "../../assets/RichTextField/I.svg";
+import Underline from "../../assets/RichTextField/U.svg";
+import UnorderedList from "../../assets/RichTextField/unordered-list.svg";
+import OrderedList from "../../assets/RichTextField/ordered-list.svg";
+
+interface RichTextFieldProps {
+  initialContent: string; // content in the stringified form of a ContentState (i.e. derived from using RichTextField)
+  // this should not be reactive! (not updated onChange)
+  // how to use: for editing pre-existing content (e.g. on editing a requestGroup's description),
+  //             pass the previous description into initialContent
```

Comment:
Move into a block comment like this

```suggestion
  /*
  content in the stringified form of a ContentState (i.e. derived from using RichTextField)
  this should not be reactive! (not updated onChange)
  how to use: for editing pre-existing content (e.g. on editing a requestGroup's description),
  pass the previous description into initialContent
  */
  initialContent: string; 
```